src/components/common/buttons/ReceiveToAddressButton.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import React from 'react'
+import { TouchableOpacity, View } from 'react-native'
+import Icon from '../view/Icon'
+import Text from '../view/Text'
+import { withStyles } from '../../../lib/styles'
+import { getDesignRelativeWidth } from '../../../lib/utils/sizes'
+
+type Props = {
+  disabled?: boolean,
+  style?: { row?: {}, icon?: {}, legendWrapper?: {}, legend?: {} },
+  onPress: any,
+  styles: any,
+  theme: any,
+}
+
+const ReceiveToAddressButton = ({ onPress, styles, theme, disabled, style = {} }: Props) => (
```

Comment:
non-scalar defaults should be defined as constants outside component
e.g.

```
const defaultStyle = {}

const ReceiveToAddressButton = ({ ..., style = defaultStyle }
```

Reply:
Done